examples/docker_aws_lambda_example/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM fedora:latest
+
+# Install pre-reqs
+RUN dnf install wget -y
+RUN dnf install gcc -y
+
+# Install librdkafka
+RUN rpm --import https://packages.confluent.io/rpm/5.4/archive.key
```

Comment:
That's quite an old version of CP, but maybe that doesn't matter.

Reply:
what is it for? i don't understand this, maybe add a comment.